packages/e2e-tests/specs/experiments/customizing-widgets.test.js

```diff
@@ -0,0 +1,430 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activatePlugin,
+	activateTheme,
+	deactivatePlugin,
+	visitAdminPage,
+	showBlockToolbar,
+	clickBlockToolbarButton,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import { find } from 'puppeteer-testing-library';
+
+describe( 'Widgets Customizer', () => {
+	beforeEach( async () => {
+		await cleanupWidgets();
+		await visitAdminPage( 'customize.php' );
+	} );
+
+	beforeAll( async () => {
+		// TODO: Ideally we can bundle our test theme directly in the repo.
+		await activateTheme( 'twentytwenty' );
+		await deactivatePlugin(
```

Comment:
Why disable this? It helps speed up tests.

Reply:
The editor doesn't load if we have this plugin enabled. It's probably a bug though.

Reply:
Ooohhh that's weird, I can repro the issue by adding `* { animation-duration: 0ms !important; animation-delay: 0s !important; transition-duration: 0s !important; transition-delay: 0s !important; }` to one of the customize widgets stylesheets.

Reply:
Maybe we are relying on `transitioned` to initialise things.

Reply:
I think [we are](https://github.com/WordPress/gutenberg/blob/add/widgets-customizer-e2e/packages/customize-widgets/src/controls/sidebar-section.js#L57)! Hmm, we should fix this as anyone using a browser extension to disable animations, or a custom stylesheet hack, won't be able to access the screen.

Reply:
Actually I think this is also a bug from core 😅. The `completeCallback` is registered within the [`_animateChangeExpanded`](https://github.com/WordPress/wordpress-develop/blob/30ff995eb0aa2660ef4b08b9e3c5f7b0126c702b/src/js/_enqueues/wp/customize/controls.js#L1596-L1607) callback, which never gets fired since it's [listening](https://github.com/WordPress/wordpress-develop/blob/30ff995eb0aa2660ef4b08b9e3c5f7b0126c702b/src/js/_enqueues/wp/customize/controls.js#L1305) the `transitionend` event.